File: OOPS/classdetails-app/src/MyClass.java

public class MyClass {

	private static MyClass object=null;
	private MyClass()
	{
			
	}
	
	public static MyClass createObject()
	{
		if(object==null)
		{
			object=new MyClass();
			return object;
		}
		return null;
	}
}

File: CSharpOOPS/customattribute/customattribute/MyClass.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using customattribute;

namespace customattribute
{
    class MyClass
    {
        [NeedForRefractoring()]
        public void add()
        {

        }
        [NeedForRefractoring()]
        public void subtract()
        {

        }
        public void multiply()
        {

        }
    }
}
